recipes/telescope/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "telescope" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [py27]
+  script: "python setup.py install --single-version-externally-managed --record=record.txt"
+
+about:
+  home: 'https://github.com/mlbendall/telescope'
+  license: MIT
```

Comment:
This is optional, but since the license file is available, you can add the following two lines:
`license_family: MIT`
`license_file: LICENSE`
here.